config/etfuturum/world.cfg

```diff
@@ -0,0 +1,192 @@
+# Configuration file
+
+##########################################################################################################
+# biomes
+#--------------------------------------------------------------------------------------------------------#
+# Biome ID and weight settings. These do not disable the respective blocks, go to blocksitems.cfg to disable their blocks. Nether biomes can currently only be enabled via experiments.cfg
+##########################################################################################################
+
+biomes {
```

Comment:
are you sure these biome ids are available?

Reply:
No. If you can give me some pointers on how to confirm, I'm happy to check.

Reply:
NEI has some data dump functions in its config. I am pretty sure biomes was one of them.

Reply:
hmm, actually...
it looks like netherDimensionProvider is false anyway as it seems to be experimental and unfinished. So the biome settings here might not matter? (not sure though)

Reply:
I don't think it matters. But I switched to biome IDs that are available based on https://docs.google.com/spreadsheets/d/1Rsz0rH9tIVJxr18b1Z6-QxOSaEKssxF7u2naQTq2Mqg/edit?gid=1251539304#gid=1251539304